src/implementations/title-escrow/reject-surrendered.ts

```diff
@@ -0,0 +1,67 @@
+import { TitleEscrowFactory, TradeTrustErc721Factory } from "@govtechsg/token-registry";
+import signale from "signale";
+import { getLogger } from "../../logger";
+import { getWallet } from "../utils/wallet";
+import { TitleEscrowSurrenderDocumentCommand } from "../../commands/title-escrow/title-escrow-command.type";
+import { dryRunMode } from "../utils/dryRun";
+import { TransactionReceipt } from "@ethersproject/providers";
+import { TradeTrustERC721 } from "@govtechsg/token-registry/dist/ts/contracts";
+import { Wallet } from "ethers";
+
+const { trace } = getLogger("token-registry:acceptSurrendered");
+
+export const retrieveLastBeneficiaryAndHolder = async (
```

Comment:
if this is a local function then there isnt a need to export it out

Reply:
removed 'export'.